resource_tracker/models.py

```diff
@@ -95,7 +100,8 @@ class ResourcePool(models.Model):
     name = models.CharField(max_length=100,
                             blank=False,
                             unique=True)
-
+    def __str__(self):
+        return self.name
```

Comment:
A space between the two def will be required to respect pep convention.
Before merging it we need to check the full file.

I'm wondering if adding a linter to the CI/CD would help.